doc/Language/create-cli.pod6

```diff
@@ -137,17 +137,17 @@ that doesn't exist, you would get the standard usage message created from
 introspection of the C<MAIN> sub:
 
 =begin code :lang<shell>
-$ perl6 frobnicate.p6 doesnotexist.dat
+$ raku frobnicate.raku doesntexist.dat
```

Comment:
Recently this was changed to doesnotexist.dat Probably better to leave it the way it was.

Reply:
oh, that was an unintended change. apologies.